### DISCUSSION TITLE
Installing HNN on Windows

### DISCUSSION THREAD

**Discussion by sarapiers** [jonescompneurolab/hnn#333] [Installation]
Hi, I am very new to all of this. I was trying to install the HNN-GUI for windows, but on the page with the instructions for it, it is recommended the " Microsoft Windows Subsystem for Linux (WSL) installation". Do I have to install a different operating system or there is also a different way to proceed? 
Here is the link of the page I am redirected to when choosing Windows in the installation page:
https://jonescompneurolab.github.io/hnn/installer/windows/wsl.html)https://jonescompneurolab.github.io/hnn/installer/windows/

Thank you 
Sara

**Comment by jasmainak** [comment]
I think it might be better if you install the new GUI as a new user ... there might not be very much support for the old GUI if you run into issues. After installing Anaconda and Neuron using the pre-compiled installers, just do:

```py
$ pip install --upgrade "hnn_core[gui] @ https://api.github.com/repos/jonescompneurolab/hnn-core/zipball/master"
```

See: https://jonescompneurolab.github.io/hnn-core/dev/index.html#installation 

cc @chenghuzi ... he will be able to help if you run into issues

**Comment by sarapiers** [comment]
Hi all, sorry for bothering you again. I followed your instructions and
installed the hnn-gui, now, I saw the tutorials available on  Tutorials -
HNN (jonescompneurolab.github.io)
<https://jonescompneurolab.github.io/hnn-tutorials/erp/erp> are on the old
GUI and I don't really know how to proceed with the one I installed, do you
have any recommendation? Also I found the page with the instructions
to Windows-native
HNN install script that allows me to install the old hnnGUI (
https://github.com/jonescompneurolab/hnn/blob/master/installer/windows/native_install.md
) , so as an alternative I can uninstall everything and install this
instead so I can follow the tutorials.

**Comment by jasmainak** [comment]
Hi @sarapiers , can you try these:

https://jonescompneurolab.github.io/hnn-core/dev/gui/tutorial_erp.html 

and 

https://jonescompneurolab.github.io/hnn-core/dev/gui/basic_gui_usage.html

These are not as detailed in terms of neuroscience as the old tutorials but hopefully gives you enough grounding to read the old tutorials and run it in the new GUI (even though the interface is slightly different). The gamma tutorial with the new GUI is also [upcoming](https://github.com/jonescompneurolab/hnn-core/pull/583) !

You can of course try to run the old GUI on a [separate environment](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) but I'm afraid it will be hard for us to fix things as we really want to move on to code that can be maintained.
